src/_data/LiveData.js

```diff
@@ -133,6 +134,6 @@ export const connect = async store => {
         api.events.on(key, data => store.dispatch(action(data)));
         api.events.on(key, () => window.console.warn);
     });
-
+    api.getResidences();
```

Comment:
@nuruddeensalihu I agree with @qingweibinary 
Please do the changes before me merging this.